src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -42,8 +56,10 @@ const TeamsTools = () => {
   const { data: allTeams, isLoading: allTeamsIsLoading, isError: allTeamsIsError, error: allTeamsError } = useAllTeams();
   const { isLoading, mutateAsync } = useCreateTeam();
   const { isLoading: allUsersIsLoading, isError: allUsersIsError, error: allUsersError, data: users } = useAllUsers();
+  const theme = useTheme();
+  const [currentDescription, setCurrentDescription] = useState('');
```

Comment:
We actually don't need this useState - most of the rest of the comments will relate to using the form without it